install/build-docker-images.sh

```diff
@@ -1,10 +1,26 @@
 echo "${_group}Building and tagging Docker images ..."
 
+declare -a SERVICES=("cron"
+                     "worker"
+                     "ingest-consumer"
+                     "post-process-forwarder-errors"
+                     "post-process-forwarder-transactions"
+                     "subscription-consumer-events"
+                     "subscription-consumer-transactions"
+                     "clickhouse"
+                     "snuba-cleanup"
+                     "snuba-transactions-cleanup"
+                     "symbolicator-cleanup"
```

Comment:
Why not `docker compose config --services`?

Reply:
Because I didn't know that was a thing, neat! I'll change this to use that :)